omod/src/main/java/org/openmrs/module/coreapps/page/controller/MarkPatientDeadPageController.java

```diff
@@ -54,7 +62,12 @@ public String post(@SpringBean("patientService") PatientService patientService,
                 patient.setCauseOfDeath(null);
             }
 
+
             patientService.savePatient(patient);
+
+            if (patient.isDead() && patient.getDeathDate() != null) {
```

Comment:
Now, the above is excellently formatted. 👍 
Can you maintain the same habit else where? 😊